flutter/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM gitpod/workspace-dart-vnc
```

Comment:
I agree with @jankeromnes in that we should just go with one Dockerfile for flutter for now.
We might split it up later if we see that there is demand and better language support (or even put dart into workspace full).

Reply:
Alrighty I'll squash it into one image